auth-web/src/components/auth/mixins/TeamManagementMixin.vue

```diff
@@ -68,14 +69,16 @@ export default class TeamManagementMixin extends Vue {
     this.modal = confirmActionDialog
     if (member.membershipStatus === MembershipStatus.Pending) {
       this.confirmActionTitle = this.$t('confirmDenyMemberTitle').toString()
-      this.confirmActionText = `Are you sure you want to deny membership to ${member.user.firstname}?`
+      this.confirmActionText = `Deny account access to ${member.user.firstname} ${member.user.lastname}?`
```

Comment:
would be nice to do the null check here and few other places.

member?.user?.firstname and member?.user?.lastname

Reply:
Don't know - that's how it was done originally. Can we do this as a separate task?

Reply:
Either way is fine, 
since its just a simple code quality improvement wont be an issue if included in this PR